examples/glfw/FlutterEmbedderGLFW.cc

```diff
@@ -82,6 +91,25 @@ void GLFWwindowSizeCallback(GLFWwindow* window, int width, int height) {
       &event);
 }
 
+const SkIRect FlutterRectToSkIRect(FlutterRect flutter_rect) {
+  SkIRect rect = {static_cast<int32_t>(flutter_rect.left),
+                  static_cast<int32_t>(flutter_rect.right),
+                  static_cast<int32_t>(flutter_rect.top),
+                  static_cast<int32_t>(flutter_rect.bottom)};
+  return rect;
+}
+
+std::array<EGLint, 4> static RectToInts(EGLDisplay display,
+                                        EGLSurface surface,
+                                        const SkIRect& rect) {
```

Comment:
Rather than taking an SkIRect, maybe just take the FlutterRect and do the conversion inline?